### ISSUE TITLE
Search not functioning after update

### ISSUE THREAD

**Issue by Jakethethird** [baumgarr/Nixnote2#384]
Version nixnote2-daily+20171030
Ubuntu 16.04

- Expected behaviour: Type word in search bar, press return and see list of just those notes with the search term in title or body of note. Individual notes have search term highlighted.
- Observed behaviour: Type word in search bar, press return, nothing happens to list of notes which remains unchanged showing all notes. However, individual notes do have search term highlighted correctly.

Basically the display of the notes in list form is not updated to show those conforming to the search term.

This worked until the last update.

Jake

**Comment by robert7** [comment]
As you maybe noticed, the original maintainer isn't available now.. It someway happened, that I took over.. There is updated version of the app ("[v.2.1](https://github.com/robert7/nixnote2/releases)") in [my fork](https://github.com/robert7/nixnote2). So far I did mainly stabilization/bugfixing and added few new features. 

I suggest try the last AppImage version, it should work OK. If not, then open issue in my fork and post log file
